### DISCUSSION TITLE
Third party stylus implementations?

### DISCUSSION THREAD

**Discussion by wbehrens-on-gh** [stylus/stylus#2728] [Q&A]
I was wanting to implement a css pre-processor for a language that I'm wanting to build a back end framework for (a hobby project) but I'm stuck because for as much as I like the syntax of stylus it doesn't seem to be defined as a spec. Unless I'm wrong in which case could someone point me to the spec docs or a different stylus implementation?

**Comment by iChenLei** [comment]
Sorry, no official spec here. It also make me confused. ( I have been maintaining this project since last year
